fhir-search/src/test/java/com/ibm/fhir/search/test/QueryParameterValueTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * (C) Copyright IBM Corp. 2018, 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.search.test;
+
+import static org.testng.Assert.assertEquals;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.search.SearchConstants.Prefix;
+import com.ibm.fhir.search.parameters.QueryParameterValue;
+
+/**
+ * This testng test class contains methods that test the QueryParameterValue class.
+ */
+public class QueryParameterValueTest extends BaseSearchTest {
+
+
+    @Test
+    public void testValueString() throws Exception {
+        QueryParameterValue testObj = new QueryParameterValue();
+        String valueString = "valueString";
+        testObj.setValueString(valueString);
+        assertEquals(testObj.getValueString(), valueString);
+        assertEquals(testObj.toString(), "valueString");
+        assertEquals(testObj.toString(), "valueString");
```

Comment:
why the duplicate lines for toString() assertion on each test?

Reply:
This was to test a fix to a previous bug where the "delim" member variable kept state across toString calls, so the 2nd call would not match the 1st call.

Reply:
got it - thanks!